src/NUnitFramework/framework/Attributes/RepeatAttribute.cs

```diff
@@ -72,7 +72,7 @@ public TestCommand Wrap(TestCommand command)
         /// <summary>
         /// The test command for the RepeatAttribute
         /// </summary>
-        public class RepeatedTestCommand : DelegatingTestCommand
+        private sealed class RepeatedTestCommand : DelegatingTestCommand
```

Comment:
This class provides no new surface area. Extending code would use `RepeatAttribute.Wrap` to obtain these instances and use composition to delegate to them.